pkg/skaffold/survey/survey.go

```diff
@@ -26,19 +26,30 @@ import (
 const (
 	Prompt = `Help improve Skaffold! Take a 10 seconds anonymous survey by running
    $skaffold survey`
+
+	Form = `Thank you for offering your feedback on skaffold. Understanding your experiences and opinions helps us make skaffold better for you and other users
+    Please take survey at https://forms.gle/BMTbGQXLWSdn7vEs6
+
+To permanently disable the survey prompt, run:
+   skaffold config set --survey --global disable-prompt true`
 )
 
 // for testing
 var (
 	isStdOut = stdOut
 )
 
-func DisplaySurveyForm(out io.Writer) {
+func DisplaySurveyPrompt(out io.Writer) {
```

Comment:
This is only used by tests?

Reply:
Yes. in Next Pr, i  will wire this up Wire up "ShouldDisplayPrompt" and "DisplaySurveyPrompt" to show the survey prompt on every skaffold run.